static/app/views/alerts/list/rules/alertRuleStatus.tsx

```diff
@@ -86,20 +89,26 @@ export default function AlertRuleStatus({rule}: Props) {
 
   return (
     <FlexCenter>
-      <IconArrow color={iconColor} direction={iconDirection} />
-      <TriggerText>
-        {`${thresholdTypeText} ${
-          rule.latestIncident || (!rule.latestIncident && !resolvedTrigger)
-            ? trigger?.alertThreshold?.toLocaleString()
-            : resolvedTrigger?.toLocaleString()
-        }`}
-        {getThresholdUnits(
-          rule.aggregate,
-          rule.comparisonDelta
-            ? AlertRuleComparisonType.CHANGE
-            : AlertRuleComparisonType.COUNT
-        )}
-      </TriggerText>
+      {rule.detectionType !== 'dynamic' ? (
```

Comment:
The explicit 'dynamic' string here is because the AlertRuleComparisonType options don't exactly match with the detection type strings. Is this okay, or how should I deal with this?

Reply:
🤔 Does it not?
I'm seeing 
```
export enum AlertRuleComparisonType {
  COUNT = 'count',
  CHANGE = 'change',
  PERCENT = 'percent',
  DYNAMIC = 'dynamic',
}
```
as the exported type?